src/UI/RowList.elm

```diff
@@ -0,0 +1,469 @@
+module UI.RowList exposing
+    ( RowList, selectList
+    , ToggleableConfig, ToggleableCover, toggleableList
+    , withOptions, withSelected
+    , SearchConfig, withSearchField, ActionConfig, withActionBar
+    , withWidth
+    , renderElement
+    )
+
+{-| `UI.RowList` is a styled searchable row list.
+The main variation (select-lists) has the capability of having one of its rows selected.
+
+The developer does the rendering of the row elements.
```

Comment:
I think this phrase is a bit confusing, maybe it would be better to say that the developer is in charge of creating the "Row" view?